src/pages/rekey-reason/rekey-reason.ts

```diff
@@ -110,6 +128,29 @@ export class RekeyReasonPage extends BasePageComponent {
       select(getReasonForRekey),
     );
 
+    this.store$.pipe(
+      select(getRekeyReasonState),
+      select(getFindUserStatus),
+    ).subscribe(
```

Comment:
We have a lot of `.subscribe`s going on in ngOnInit, can we not roll them into the merged subscription in `this.subscription`? Any operations that need to be done can be done as a `map` after the `select` operator. This will also ensure we unsubscribe when the page is exited.